### PR TITLE
tullia jobs: use generic gitrev for PRs

### PR DESCRIPTION
**Pull request by jbgi**
to avoid unnecessary builds of big artifact, like dockerImages.
 Goal: bors ci build should not build anything if branch is up-to-date
 and ci pass.

 Some other minor clean-ups.

### CODE REVIEW

File: nix/tullia.nix [hunk 1]
```diff
@@ -104,7 +104,7 @@ rec {
         #lib.io.github_push
         #input: "${ciInputName}"
         #repo: "${repository}"
-        #branch: "master|staging|trying"
+        #branch: "master|release/1.35"
```
Comment (dermetfan) [inline nix/tullia.nix:107]: This is a regex, you can match all releases:
```suggestion
        #branch: "master|release/.*"
```

File: flake.nix [hunk 2]
```diff
@@ -143,252 +143,250 @@
...
+        checks = flattenTree project.checks //
+          # Linux only checks:
+          (optionalAttrs hostPlatform.isLinux (
+            prefixNamesWith "nixosTests/" (mapAttrs (_: v: v.${system} or v) nixosTests)
+          ))
+          # checks run on default system only;
+          // (optionalAttrs (system == defaultSystem) {
+          #hlint = pkgs.callPackage pkgs.hlintCheck {
+          #  inherit (project.args) src;
+          #};
```
Comment (dermetfan) [inline flake.nix:190]: Forgot to uncomment?

Comment (jbgi) [reply flake.nix:190]: ah right.

**Review by dermetfan — APPROVED** [review]

**Comment by jbgi** [comment]
bors merge

**Comment by jbgi** [comment]
bors merge

**Comment by jbgi** [comment]
bors retry

**Comment by jbgi** [comment]
bors retry

**Comment by jbgi** [comment]
bors retry

**Pull request merged**